CODEOWNERS

```diff
@@ -1 +1 @@
-* @launchdarkly/squad-developer-flow
\ No newline at end of file
+* @kinyoklion
```

Comment:
I suppose that will be handy. Should we both be in here, in the unlikely case I open a PR? :-)